kitchen.global.yml

```diff
@@ -7,3 +7,4 @@ provisioner:
       region: us-east-1
       dcv:
         server: {}
+      parallelcluster-node-version: '3.4.1'
```

Comment:
should we add it to version-bump?

Reply:
No, not this way (take a look at the message on the commit). We will need to think more about it, this is temporary.